src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/tests/List.test.js

```diff
@@ -387,6 +397,46 @@ test('Render the adapter in disabled state', () => {
     ).toMatchSnapshot();
 });
 
+test('Render the adapter with filters', () => {
```

Comment:
`with filterable fields`?

Reply:
Have overlooked that one, gonna fix it when doing the other missing changes, after you have commented again.

Reply:
Actually I think the name is correct... it does not only render with some filterable fields, it renders with some filters really applied